[Docs] Add link to dependencies page - v3 (#797)
elastic/docs-content: deploy-manage/deploy/self-managed/install-kibana.md

## Installation order

:::{include} /deploy-manage/deploy/_snippets/installation-order.md
:::

## Third-party dependencies

> Refer to [{{stack}} third-party dependencies](https://artifacts.elastic.co/reports/dependencies/dependencies-current.html) for a list of the external dependencies you can leverage.

Reviewer: why is the person "leveraging" the dependencies? wouldn't the dependencies just be needed for functioning of es?

Reviewer 2: Yep let's stick to the original wording
```suggestion
Refer to [{{stack}} third-party dependencies](https://artifacts.elastic.co/reports/dependencies/dependencies-current.html) for the complete list of dependencies.
```

Author: You're both right, the original wording works. I was overthinking it. Updated wording and turned the text into a snippet to include on both pages. Thanks!
